sdk/examples/how_tos/nfts/mint_nft.rs

```diff
@@ -0,0 +1,96 @@
+// Copyright 2022 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+//! In this example we will mint a native token.
+//! Rename `.env.example` to `.env` first.
+//!
+//! `cargo run --example mint_nft --release`
+
+use iota_sdk::{
+    types::block::{
+        address::Bech32Address,
+        output::{
+            feature::{IssuerFeature, SenderFeature},
+            unlock_condition::AddressUnlockCondition,
+            NftId, NftOutputBuilder,
+        },
+    },
+    wallet::{MintNftParams, Result, Wallet},
+};
+
+#[tokio::main]
+async fn main() -> Result<()> {
+    // This example uses secrets in environment variables for simplicity which should not be done in production.
+    dotenvy::dotenv().ok();
+
+    // Create the wallet
+    let wallet = Wallet::builder().finish().await?;
+
+    // Get the account we generated with `01_create_wallet`
+    let account = wallet.get_account("Alice").await?;
+
+    // Set the stronghold password
+    wallet
+        .set_stronghold_password(&std::env::var("STRONGHOLD_PASSWORD").unwrap())
+        .await?;
+
+    let nft_options = vec![MintNftParams {
+        address: Some(Bech32Address::try_from_str(
+            "rms1qpszqzadsym6wpppd6z037dvlejmjuke7s24hm95s9fg9vpua7vluaw60xu",
+        )?),
+        sender: Some(Bech32Address::try_from_str(
+            "rms1qpllaj0pyveqfkwxmnngz2c488hfdtmfrj3wfkgxtk4gtyrax0jaxzt70zy",
+        )?),
+        metadata: Some(b"some NFT metadata".to_vec()),
+        tag: Some(b"some NFT tag".to_vec()),
+        issuer: Some(Bech32Address::try_from_str(
+            "rms1qpllaj0pyveqfkwxmnngz2c488hfdtmfrj3wfkgxtk4gtyrax0jaxzt70zy",
+        )?),
```

Comment:
Sender and issuer will not work with a different mnemonic, should probably also just use the first account address as in the nodejs examples